theatre/studio/src/panels/OutlinePanel/BaseItem.tsx

```diff
@@ -132,6 +127,7 @@ const BaseItem: React.FC<{
   labelDecoration?: React.ReactNode
 }> = ({label, children, depth, select, selectionStatus, labelDecoration}) => {
   const canContainChildren = children !== undefined
+  const [isOpen, setIsOpen] = useState(true)
```

Comment:
I think ideally this would be stored in the studio data. Happy to merge this PR and add that in follow-up PR though.